viewer/tsconfig.test.json

```diff
@@ -0,0 +1,35 @@
+{
+  "compilerOptions": {
+    "incremental": true,
+    "target": "ES2019",
+    "module": "esnext",
+    "declaration": true,
+    "declarationDir": "dist",
+    "outDir": "dist-ts",
+    "rootDir": "src",
+    "esModuleInterop": true,
+    "downlevelIteration": true,
+    "sourceMap": true,
+    "moduleResolution": "node",
+    "strict": true,
+    "stripInternal": true,
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
+    "typeRoots": [
+        "node_modules/@types",
+        "src/@types"
+    ],
+    "lib": [
+        "dom",
+        "esnext"
+    ],
+
+    "baseUrl": "./src/",
+    "paths": {
+      "@/*": ["*"]
+    }
+  },
+  // include is also used for eslint to understand which files should be linted
```

Comment:
That shouldn't work too.

Reply:
Sorry, I shouldn't review anything until morning coffee. Yep, json doesn't allow comments, but tsconfig.json does.